core/common/fs_mobile.go

```diff
@@ -0,0 +1,35 @@
+//go:build ios || android
+// +build ios,android
+
+package common
+
+import (
+	"io/fs"
+	"os"
+)
+
+// FS An FS provides access to a hierarchical file system.
+type FS interface {
```

Comment:
it seems be same as fs.go completely? is it possible to reuse it instead of duplicated one?